packages/shared/providers.yaml

```diff
@@ -539,6 +539,39 @@ contentstack:
     authorization_url: https://${connectionConfig.subdomain}.contentstack.com/apps/${connectionConfig.appId}/authorize
     token_url: https://${connectionConfig.subdomain}.contentstack.com/apps-api/apps/token
     docs: https://docs.nango.dev/integrations/all/contentstack
+coros:
+    categories:
+        - sports
+    auth_mode: OAUTH2
+    authorization_url: https://open.coros.com/oauth2/authorize
+    token_url: https://open.coros.com/oauth2/accesstoken
+    refresh_url: https://open.coros.com/oauth2/refresh-token
+    authorization_params:
+        response_type: code
+    token_response_metadata:
+        - openId
+    token_params:
+        grant_type: authorization_code
+    refresh_params:
+        grant_type: refresh_token
+    proxy:
+        base_url: https://open.coros.com
+coros-sandbox:
+    auth_mode: OAUTH2
+    authorization_url: https://opentest.coros.com/oauth2/authorize
+    token_url: https://opentest.coros.com/oauth2/accesstoken
+    refresh_url: https://opentest.coros.com/oauth2/refresh-token
+    # scope_separator: ' '
```

Comment:
You are right, there is no mention of scopes either during authorization or during the code exchange for an `access_token` step in their documentation. I think it would be safe to remove this line.

Reply:
Done!